.dockerignore

```diff
@@ -0,0 +1,9 @@
+# Ignore everything
+**
+
+# Allow files and directories
```

Comment:
Ah. docker would create context based on the entire repo during the build process. With .dockerignore it would only create context based on what is useful. Is that how it is supposed to work?

Reply:
This is what i see on the Jenkins  with .dockerignore 

[2023-04-03T22:34:39.652Z] + docker build -t migraphxlib .
[2023-04-03T22:34:39.652Z] Sending build context to Docker daemon  41.98kB


Without .dockerignore : 
[2023-03-31T22:21:38.255Z] + docker build -t migraphxlib .
[2023-03-31T22:21:38.823Z] Sending build context to Docker daemon  76.06MB

Build context is much smaller.